builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -67,6 +71,24 @@ steps:
     projects: '${{ parameters.solution }}'
     arguments: '--configuration ${{ parameters.configuration }} -p:GeneratePackageOnBuild=false -p:Version=${{ parameters.binariesVersion }}'
 
+- task: CopyFiles@2
+  displayName: 'Copy Sql extension dll to Azure Functions extension bundle for Windows'
+  inputs:
+    sourceFolder: $(Build.SourcesDirectory)\src\bin\${{ parameters.configuration }}\netstandard2.0
```

Comment:
I didn't think these tasks cared about path separators - they were able to take in either. Is that not actually the case?

Reply:
Oh I thought I saw somewhere that they do but I can verify.

Reply:
Verified that we can just use one task.